src/Survey/Default/config/index.ts

```diff
@@ -172,34 +208,21 @@ const survey: Survey = {
               info: 'How many individuals of this species did you see?',
               inputProps: { max: 500 },
             },
-            {
-              set: (value: string, model: AppOccurrence) =>
-                Object.assign(model.attrs, {
-                  number: undefined,
-                  'number-ranges': value,
-                }),
-              get: (model: AppOccurrence) => model.attrs['number-ranges'],
-              onChange: () => window.history.back(),
-              input: 'radio',
-              inputProps: { options: numberOptions },
-            },
           ],
         },
-        remote: { id: 16 },
+        remote: { id: 93 },
       },
 
-      'number-ranges': { remote: { id: 523, values: numberOptions } },
-
       stage: {
-        menuProps: { icon: progressIcon },
+        menuProps: { icon: fingerprintIcon },
         pageProps: {
           attrProps: {
             input: 'radio',
-            info: 'Please pick the life stage.',
+            info: 'Please pick the life stage and sex of the organism, if recorded.',
             inputProps: { options: stageOptions },
           },
         },
-        remote: { id: 106, values: stageOptions },
+        remote: { id: 218, values: stageOptions },
```

Comment:
I changed some of the remote IDs in this file to match the ID's used in ORKS. See a reference here to the v5 ID https://github.com/ERCCIS/orks-app/blob/master/src/common/config/surveys/default.js#L141